sechub-pds-solutions/cloc/01-start-single-docker-compose.sh

```diff
@@ -17,4 +17,4 @@ export BUILDKIT_PROGRESS=plain
 export DOCKER_BUILDKIT=1
```

Comment:
Those changes are an improvement. However, they are affecting only the pds-cloc solution. I would appreciate if you could create an issue and implement it in a separate pull-request. Simply, to keep this limited to mostly the tutorial.

Please do this for all `pds-cloc` scripts.